fr-portfolio.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en-gb">
```

Comment:
There's a lot of inconsistent indenting in this file that needs sorting out.
Use spaces to indent, btw, not tabs. It'll help keep it consistent no matter what editor is used.

Reply:
I have gone over all the indenting using spacebar hopefully it is in a better place now